MapboxCoreNavigation/PortableRouteController.swift

```diff
@@ -1,14 +1,18 @@
 import Foundation
+import CoreLocation
 import MapboxNavigationNative
+import MapboxMobileEvents
 import MapboxDirections
+import Polyline
+import Turf
 
 
 @objc(MBPortableRouteController)
-open class PortableRouteController: RouteController {
+open class PortableRouteController: NSObject {
```

Comment:
This is an exciting change. What is going to happen with the existing `RouteController` as a part of this PR?

Reply:
Ideally, we would just delete it, but I assume we have to deprecate it. 🤔